src/backend/users/tests.py

```diff
@@ -80,7 +80,7 @@ def test_counselor_group_user_auth_permissions(self):
         self.assertNotIn(
             other_counselor_user.username, content, "Got unexpected content: %s" % content
         )
-        self.assertNotIn("Add", content, "Got unexpected content: %s" % content)
+        self.assertNotIn("Add User", content, "Got unexpected content: %s" % content)
```

Comment:
I believe this was to test that the counselor user should not be able to add user. There is an "Add link" button on the admin page after the upgrade, but no "Add user". This is to make it more explicit. But please let me know if this is not a right understanding.